### DISCUSSION TITLE
What should be our Jest mock to test react-native-webview components

### DISCUSSION THREAD

**Discussion by eppisapiafsl** [react-native-webview/react-native-webview#3037] [Q&A]
Env
```
"react-native": 0.71.12,
"react-native-webview": 13.2.2,
"jest": "29.5.0"

```

I just upgrade from 11.26.1 to 13.2.2, all good till I run the unit testing
![image](https://github.com/react-native-webview/react-native-webview/assets/82107248/90bd8cf8-d4b4-4cd0-9957-d9852b991fcb)

Following the fix at https://github.com/react-native-webview/react-native-webview/pull/2686 

We need to mock the library, I was able to fix most of the test by declaring an empty View component and mocking a few function 

But still facing issues on testing, what would be the proper mock to implement?

**Comment by angelo-hub** [comment]
bump on this!
